dodo.py

```diff
@@ -399,6 +399,48 @@ def task_yarn():
         ],
     }
 
+def task_perf():
+    '''
+    run locustio performance tests
+    '''
+    ID='fake-id'
+    KEY='fake-key'
+    PP='.'
+    FLASK_PORT=5000
+    PROFILING="True"
+    cmd = f'env {envs(LOCAL_FLASK_PORT=FLASK_PORT, AWS_ACCESS_KEY_ID=ID, AWS_SECRET_ACCESS_KEY=KEY, PYTHONPATH=PP, PROFILING_ENABLED=PROFILING)} {PYTHON3} subhub/app.py'
+    return {
+        'task_dep':[
+            'check',
+            'stripe',
+            'venv',
+            'dynalite:start'
+        ],
+        'actions':[
+            f'{PYTHON3} -m setup develop',
+            'echo $PATH',
+            LongRunning(f'nohup {cmd} > /dev/null &'),
+            f'locust -f subhub/tests/performance/locustfile.py --host=http://localhost:{FLASK_PORT}'
+        ]
+    }
+
+def task_remote_perf():
+    '''
+    run locustio performance tests
+    '''
+    REMOTE_HOST='https://qa.fxa.mozilla-subhub.app'
```

Comment:
I happily extract this into a configuration as other variables should we suspect to leverage this in other environments in the future.